pkg/synthetictests/disruption.go

```diff
@@ -18,6 +18,14 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 )
 
+// getRoundedAllowedDisruption is used to calculate allowed disruption based on historical data.
+// We found using 99 percentile of historical data can cause lots of noises. This experiment is to
+// allow a little over 10% when the number is big, while keep a minimum of 10s while the number is
+// small.
+func getRoundedAllowedDisruption(allowedDisruption time.Duration) time.Duration {
+	return (allowedDisruption*11/10 + 10).Round(time.Second)
```

Comment:
Suggest allowedDisruption*1.1 + 10 for clarity.

Reply:
allowedDisruption is time.Duration. So this suggestion does not work. I am not sure it will be as clear as we hope if I add some cast to make it work. Maybe allowedDisruption*110/100 + 10?

Reply:
Ok no worries then, lets just leave as is, we all had a good look this morning.